utils/registerUser.mjs

```diff
@@ -0,0 +1,17 @@
+import { doFetch } from "./doFetch.mjs";
+import { REGISTER_URL } from "../shared/constants.mjs";
+export async function registerUser(name, userName, email, password){
+    console.log('register user');
+    // Build formData object.
+    const regForm = {
+        name,
+        userName,
+        email,
+        password
+    };
+    await doFetch(REGISTER_URL, false, {
+    method: 'POST',
+    body: JSON.stringify(regForm)
+});
+
+}
```

Comment:
1) A `try/catch` is missing here in this `registerUser()` function.

2) Usage of `sessionStorage.setItem()` is also missed here to store the `email` field.